xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-wikimacro/xwiki-platform-rendering-wikimacro-store/src/main/java/org/xwiki/rendering/wikimacro/internal/DefaultWikiMacroFactory.java

```diff
@@ -365,4 +372,9 @@ public boolean isAllowed(DocumentReference documentReference, WikiMacroVisibilit
 
         return isAllowed;
     }
+
+    private List<String> getDefaultCategories(BaseObject macroDefinition)
+    {
+        return (List<String>) macroDefinition.getListValue(MACRO_DEFAULT_CATEGORIES_PROPERTY);
```

Comment:
Would using `#getSetValue` make more sense here ?

Reply:
Good catch thanks, I'll change that.